src/Servers/Kestrel/Core/src/KestrelServerLimits.cs

```diff
@@ -42,18 +42,26 @@ public class KestrelServerLimits
     private long? _maxConcurrentConnections;
     private long? _maxConcurrentUpgradedConnections;
 
+#pragma warning disable CS1574 // XML comment has cref attribute that could not be resolved
     /// <summary>
     /// Gets or sets the maximum size of the response buffer before write
     /// calls begin to block or return tasks that don't complete until the
     /// buffer size drops below the configured limit.
     /// Defaults to 65,536 bytes (64 KB).
+    /// <para>
+    /// For limits to work correctly, this needs to be consistent with
```

Comment:
Do we want to call out what it means for limits to not work correctly? In this case it means you might get a truncated response right?

Reply:
I think it's truncated in one direction and not adequately limited in the other.  TBH, I still think it's silly to add public docs to answer a question no one's asking and then remove them in the next release (when, presumably, we'll have revamped the limits to actually make sense).

Reply:
Elaborated.